### DISCUSSION TITLE
limitation/recommendation

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#477] [Questions] [answered]
For how many users can an AIO instance be used?  Is there any limitation or is it infinitely scalable and depends only on the power of the server?  Or is there a recommendation (Possibly for Nextcloud itself)?

**Comment by szaimen** [comment]
Moving to discussions

**Comment by szaimen** [comment, marked as answer]
Afaik, AIO should scale linearly with the given hardware. However, at some point, (when the instance becomes very big), you should probably think about clustering your instance, adding a load balancer, network filesystem apart from other things which AIO cannot provide.
